api/models/post/createPost.js

```diff
@@ -33,30 +33,30 @@ export function afterCreatingPost (post, opts) {
 
     // Add media, if any
     // redux version
-    opts.imageUrl && Media.createForPost({
+    opts.imageUrl && Media.createForSubject({
       postId: post.id, type: 'image', url: opts.imageUrl
     }, trx),
 
-    opts.videoUrl && Media.createForPost({
-      postId: post.id, type: 'video', url: opts.videoUrl
-    }, trx),
-
     // evo version
     opts.imageUrls && Promise.map(opts.imageUrls, (url, i) =>
-      Media.createForPost({
+      Media.createForSubject({
         postId: post.id, type: 'image', url, position: i
       }, trx)),
 
     // evo version
     opts.fileUrls && Promise.map(opts.fileUrls, (url, i) =>
-      Media.createForPost({
+      Media.createForSubject({
         postId: post.id, type: 'file', url, position: i
       }, trx)),
 
     opts.children && updateChildren(post, opts.children, trx),
 
-    opts.docs && Promise.map(opts.docs, doc => Media.createDoc(post.id, doc, trx)),
+    // google doc / video not currently used in evo
```

Comment:
we do really want video support though

Reply:
Well yeah, but that's a whole other thing requiring a transcoding service and such.

Reply:
yes of course